### ISSUE TITLE
[Captions] Include quality checks?

### ISSUE THREAD

**Issue by iadawn** [w3c/wai-easy-checks#152]
Should the [checking captions](https://deploy-preview-131--wai-easychecks.netlify.app/test-evaluate/easy-checks/captions/#checking-captions) section include information checking the quality of captions?

More? Less? Different checks? None at all?

**Comment by JediLin** [comment]
I think it should.
I'm not sure if this section should include information on translated captions.

I'd like to suggest a little change too:

Current wording:
- Check if other important sounds, like applause or thunder rumbling, are included.

Suggested revision:
- Check if other important sounds, like applause or thunder rumbling, are included in default captions or additional SDH captions.

**Comment by iadawn** [comment]
@JediLin What is SDH?

**Comment by iadawn** [comment]
I think the idea of 'default captions' and 'SDH' goes beyond the 'easy' requirement. So unless you feel strongly about this, I am going to propose not adding this distinction.

**Comment by JediLin** [comment]
In this case, I'd like to propose deleting this entire item:
- Check if other important sounds, like applause or thunder rumbling, are included.

Or make it softer, such as:
- Other important sounds, like applause or thunder rumbling, might be included.

**Comment by iadawn** [comment]
@JediLin I am not sure of the rationale for the change. It seems that you are suggesting these are secondary things that might be included. However, is that counter to the opening 'Other *important* sounds'?

Happy to hear your thoughts

**Comment by JediLin** [comment]
In my general TV programs (online videos, streaming service programs) watching experience, videos with closed captions:
- almost all provide dialog caption in original language, labeled with language name (ex: `English`)
- some provide translated dialog caption in other languages, labeled with language name (ex: `Spanish`)
- few provide dialog and import sounds caption in original language, labeled with language name and "SDH" (ex: `English (SDH)`)

When putting people with low accessibility knowledge as target audience in mind, I think they will mainly encounter dialog-only captions.

And in my field experience with actual users with hearing disabilities, some prefer SDH captions while others prefer non-SDH ones for SDH captions do interfere visual presentation more. Usually I'll suggest content authors to provide both versions of caption.

I do think spoken content is more important than other (still important) sounds here.

**Comment by AndrewArch** [comment]
In Australia, all TV captions that I've observed (and my wife usually turns them on) include background sounds like [bird chirping] or [door creaking] . Looking at some streaming TV channels, in particular [closed captions in SBS On Demand](https://help.sbs.com.au/hc/en-au/articles/360002032376-How-do-I-enable-closed-captions-in-SBS-On-Demand-), I find that 'English CC' is recommended on all platforms except iOS mobile where 'English SDH' is recommended.

At Intopia, we always recommend including important sounds in the standard CC - IMHO most corporates or government agencies are not going to produce 2 sets.

**Comment by JediLin** [comment]
Indeed, different regions will have different practices/preferences. We might like to consider that.

In Taiwan, for some political and historical reasons, the main stream is open caption and is dialog-only (without speaker-indicator) for decades.
In fact lots of Taiwanese YouTuber still make YouTube videos with open caption for viewers, and closed caption (again, dialog-only) just for SEO or YouTube recommendation algorithm.

Fun fact: In Taiwan, TV captions (open caption) in early ages are all right-to-left, later left-to-right.
